src/App.js

```diff
@@ -22,12 +29,18 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  value={this.state.query}
+                  onChange={(event) => {
+                    this.setState({ query: event.target.value });
```

Comment:
Let's create a method like `handleChange` and change the state in it. Here 'onChange' we will just call it

Reply:
done